packages/bpk-component-segmented-control/src/BpkSegmentedControl.module.scss

```diff
@@ -102,3 +102,19 @@
 .bpk-segmented-control[class*='rightOfOption'] {
   border-inline-start: none;
 }
+
+.bpk-segmented-control:first-child {
+  border-top-left-radius: tokens.$bpk-border-radius-sm;
+  border-bottom-left-radius: tokens.$bpk-border-radius-sm;
+}
+
+.bpk-segmented-control:last-child {
+  border-top-right-radius: tokens.$bpk-border-radius-sm;
+  border-bottom-right-radius: tokens.$bpk-border-radius-sm;
+}
+
+.bpk-segmented-control:focus {
+  border: (tokens.$bpk-one-pixel-rem * 2) solid tokens.$bpk-text-link-day;
+  outline: none;
+  outline-offset: (tokens.$bpk-one-pixel-rem * 2);
```

Comment:
We have `bpk-focus-indicator()` to support this :) 

https://github.com/Skyscanner/backpack/blob/main/packages/bpk-mixins/_utils.scss#L34-L37

Reply:
You will just have to add the `outline: none` yourself

Reply:
I was getting errors when trying to use it that it was undefined even though utils is imported at the top of the file. Let me try again

Reply:
The use should look something like 

`@include utils.bpk-focus-indicator`

Reply:
That doesn't work as the mixin sets the outline to be `(tokens.$bpk-one-pixel-rem * 2) solid tokens.$bpk-text-link-day;` where as we need it to be the border here

Reply:
The outline offset here wasn't actually doing anything